src/components/EventTypeFilters.vue

```diff
@@ -1,12 +1,22 @@
 <template>
-  <div class="event-type-filters">
-    <h5 class="event-filters-title">Type of event</h5>
-    <div v-for="(label, type) in eventTypes">
-      <input type="checkbox"
-        class="no-margin-checkbox"
-        :checked="selectedEventTypes.includes(type)" 
-        @change="updateSelectedEventTypes(type, $event)"
-      /> {{label}}
+  <div>
+    <h5 v-if="showTitle" class="filter-events-title">Type of event</h5>
+    <div 
+      class="filter-events-item" 
+      v-for="(label, type) in eventTypes"
+    >
+      <div class="filter-events-checkbox">
```

Comment:
Hrmm...IIRC there is some limitation or x-browser styling 💩  situation that makes it more or less necessary to hide the real checkbox element rather than style it, for CSS-only custom checkboxes. I'll look into this though, and this is a good point about accessibility, the actual text should be in the label. If nothing else maybe I could just drop the way the label is being used and create an additional element to represent the styled checkbox, but keep the sibling CSS selector.